apps/src/javalab/JavalabEditor.jsx

```diff
@@ -167,6 +134,10 @@ class JavalabEditor extends React.Component {
       });
     }
 
+    if (prevProps.height !== this.props.height) {
+      styles.editor.height = `${this.props.height}px`;
```

Comment:
Rather than editing the styles object, you can use this.props.height directly (I'll leave a note below with details). This will help with the maintainability. Reason: given that the styles object is `const` I would have been surprised to see it changed here if I was looking at this code out of context.

Reply:
Oh nice- took this out and updated according to the below note. Thanks!